app/views/miq_action/show.html.haml

```diff
@@ -183,6 +183,29 @@
         .col-md-8
           = h(@record.options[:scan_item_set_name])
     %hr
+  - when "run_ansible_playbook"
+    %h3
+      = _("Run Ansible Playbook")
+    .form-horizontal
+      .form-group
+        %label.control-label.col-md-2
+          = _("Playbook Catalog Item")
+        .col-md-8
+          = h(ServiceTemplateAnsiblePlaybook.order(:name).pluck(:name, :id).find { |(_, id)| id == @record.options[:service_template_id] }[0])
```

Comment:
Try to store this to variable in ruby/javascript and refer it here or may be store it in @record.options[:service_template_name]?